utils/test_harness/include/test_harness/test_harness_memory.hpp

```diff
@@ -29,6 +28,11 @@ const auto memory_allocation_sizes =
 const auto memory_allocation_alignments = ::testing::Values(
```

Comment:
Considering that we separate small and large now, should be remove this regular alignment values here?

Reply:
These are used for a few other CTS as well

Reply:
Thanks, that's good enough, so please Resolve this conversation to proceed further.